src/Search/FlagrowIOSearcher.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Flagrow\Bazaar\Search;
+
+use Flagrow\Bazaar\Extensions\Extension;
+use Flarum\Core\Search\SearchResults;
+use Flarum\Extension\ExtensionManager;
+use Flarum\Settings\SettingsRepositoryInterface;
+use GuzzleHttp\Client;
+use Illuminate\Database\Eloquent\Collection;
+
+class FlagrowIOSearcher extends AbstractExtensionSearcher
```

Comment:
I'm against using the vendor in the classname again, what was the reason to do so?